src/graphql/resolvers/car.ts

```diff
@@ -0,0 +1,97 @@
+import { Query, Resolver, Arg, Mutation } from "type-graphql";
+import Car, { CarInput } from "../schemas/car";
+import CarModel from "../../db/models/car";
+import { v4 as uuidv4 } from "uuid";
+import { Op } from "sequelize";
+
+@Resolver((of) => Car)
+export default class {
+
+  createCar(@Arg("model") model: CarInput) {
+    return CarModel.create({ ...model, id: uuidv4()});
+  }
+
+  @Mutation(() => Car)
+  async updateCar(@Arg("name") name: string, @Arg("id") id: string) {
+    const CarToUpdate = await CarModel.findByPk(id);
+    return CarToUpdate.update({ name });
+  }
+
+  @Mutation(() => Car)
+  async upsertCar(@Arg("model") model: CarInput) {
+    if (!model.id) {
+      model.id = uuidv4();
+      model.createdAt = new Date();
+    }
+    const existingCar = await CarModel.findByPk(model.id);
+    if (existingCar) {
+      model.createdAt = existingCar.createdAt;
+    }
+    await CarModel.upsert({
+      id: model.id,
+      name: model.name === undefined ? existingCar.name : model.name,
+    });
+    return CarModel.findByPk(model.id);
+  }
+
+  @Mutation(() => Boolean)
+  async deleteCar(@Arg("id") id: string) {
+    const deleted = await CarModel.destroy({ where: { id } });
```

Comment:
instead of returning true or false, you should return the id if the row is successfully deleted, and throw an error if something goes wrong.  Returning false won't give the FE any idea of what went wrong.

I recommend wrapping these mutations in a try catch block
<img width="407" alt="Screen Shot 2021-01-25 at 10 33 02 AM" src="https://user-images.githubusercontent.com/31671880/105734876-b49cb000-5ef8-11eb-9e1c-65582f12b62c.png">